docs/user-guide/compute-resources.md

```diff
@@ -266,13 +281,13 @@ The current system only allows resource quantities to be specified on a containe
 It is planned to improve accounting for resources which are shared by all containers in a pod,
 such as [EmptyDir volumes](volumes.md#emptydir).
 
-The current system only supports container limits for CPU and Memory.
+The current system only supports container requests and limits for CPU and Memory.
 It is planned to add new resource types, including a node disk space
 resource, and a framework for adding custom [resource types](../design/resources.md#resource-types).
 
-The current system does not facilitate overcommitment of resources because resources reserved
-with container limits are assured.  It is planned to support multiple levels of [Quality of
-Service](http://issue.k8s.io/168).
+In future, the system will be able to facilitate overcommitment of resources by supporting multiple
```

Comment:
This is already implemented. So I would replace this paragraph with this one sentence:
Kubernetes supports overcommitment of resources by supporting multiple levels of [Quality of Service](../proposals/resource-qos.md).

(We really should move the doc out of "proposals" but for now that's where it is...)